api/schemas/report.json

```diff
@@ -114,6 +114,11 @@
             ]
         },
 
+        "master": {
+            "description": "The name of the master that was used to compile the catalog. If failover occurred, this will hold the first master successfully contacted. If this run had no master (e.g. a `puppet apply` run), this field will be blank",
```

Comment:
No wait, it should always populate I think. Maybe the description is misleading. I have to investigate if it actually *does*, but it seems to me like it *should*, unless doing a run with no master.